Intersect (Core)/Enums/TagsPosition.cs

```diff
@@ -0,0 +1,25 @@
+namespace Intersect.Enums
+{
+	/// <summary>
+	/// Enum used for switching Npc Tags Position.
+	/// </summary>
+	public enum NpcTagPos
```

Comment:
Should just be called `TagPosition` since it is generic.

Reply:
👍🏽  !

```
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Intersect.Enums {
	/// <summary>
	/// Enum used for switching Tags Position.
	/// </summary>
	[JsonConverter(typeof(StringEnumConverter))]
	public enum TagPos {
		Above,
		Under,
		Prefix,
		Suffix
	}
}
```